docs/intake_regionmask.rst

```diff
@@ -0,0 +1,87 @@
+############################
+Using regionmask with intake
+############################
+
+Regions from geopandas shapefiles can be also pre-defined in a yaml file, which can be
+easily shared. This relies on ``intake_geopandas`` and accepts ``regionmask_kwargs``,
+which are passed to ``regionmask.from_geopandas``.
+If you set ``use_fsspec=True`` and use ``simplecache::`` in the url, the shapefile is
+cached locally.
+
+.. ipython:: python
+    :suppress:
+
+    # Use defaults so we don't get gridlines in generated docs
+    import matplotlib as mpl
+    mpl.rcdefaults()
+    mpl.use('Agg')
+
+    # cut border when saving (for maps)
+    mpl.rcParams["savefig.bbox"] = "tight"
+
+You need to install intake_geopandas, which combines geopandas and intake, see
+https://intake.readthedocs.io/en/latest/.
+
+.. ipython:: python
+
+    import intake_geopandas
+    import intake
+    # open a pre-defined remote or local catalog yaml file
+    url = 'https://raw.githubusercontent.com/aaronspring/remote_climate_data/master/master.yaml'
+    cat = intake.open_catalog(url)
+    # access data from remote source
+    region = cat.regionmask.MEOW.read()
+    print(region)
+
+    region.plot(add_label=False)
+
+    @savefig plotting_MEOW.png width=100%
+    plt.tight_layout()
+
+
+Build your own catalog
+======================
+
+Let's explore the Marine Ecoregions Of the World (MEOW) data set, which is a
+biogeographic classification of the world's coasts and shelves.
+To create a catalog we use the syntax described in
+`intake <https://intake.readthedocs.io/en/latest/catalog.html#yaml-format>`_.
+
+.. ipython:: python
+
+    %%writefile test_cat.yml
```

Comment:
this is another problem. these magic commands work in notebooks. But I couldnt find a way how to write a new file here in the rst file. is bash somehow possible?

```
#!/bin/bash

cat > FILE.txt <<EOF

info code info 
info code info
info code info

EOF 
```

Reply:
should I rather provide a notebook? new nice for git but at least this should work...

Reply:
not the problem. problem caused here from remote: 
```python
>>> import intake
>>> cat = intake.open_catalog('https://raw.githubusercontent.com/aaronspring/remote_climate_data/master/master.yaml')
Traceback (most recent call last):
```
I will have to fix that. thought this was very stable but isnt.

Reply:
works now.